documentation/examples/todoist/todoist.ts

```diff
@@ -1,100 +1,404 @@
-// This import statement gives you access to all parts of the Coda Packs SDK.
 import * as coda from "@codahq/packs-sdk";
 
-// This line creates your new Pack.
+
+// Constants.
+
+const ProjectUrlPatterns: RegExp[] = [
+  new RegExp("^https://todoist.com/app/project/([0-9]+)$"),
```

Comment:
Any reason to use `new RegExp()` instead of the slash notation? Is this part of the style guide?

Reply:
No, it was just to avoid having to escape so many forward slashes in the URL pattern.